engines/dfc_provider/spec/controllers/dfc_provider/api/catalog_items_controller_spec.rb

```diff
@@ -96,4 +99,53 @@
       end
     end
   end
+
+  describe('.show') do
```

Comment:
No need for braces here, and the dot notation isn't used elsewhere in OFN specs. I would use: 
```ruby
describe '#show' do`
```